techniques/general/G131.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Providing descriptive labels</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></link></head><body><h1>Providing descriptive labels</h1><section class="meta"><p class="id">ID: G131</p><p class="technology">Technology: general</p><p class="type">Type: Technique</p></section><section id="applicability"><h2>When to Use</h2>
       <p>All technologies.</p>
    </section><section id="description"><h2>Description</h2>
-      <p>The objective of this technique is to ensure that the label for any interactive component within Web content makes the component's purpose clear. Using the appropriate technology-specific techniques for technologies for associating labels with interactive controls allows assistive technology to recognize the label and present it to the user, therefore allowing the user to identify the purpose of the control.The label may also be used to include text or a text symbol indicating that input is required.
+      <p>The objective of this technique is to ensure that the visible label for any interactive component within Web content makes the component's purpose clear. Using the appropriate technology-specific techniques for technologies for associating labels with interactive controls allows assistive technology to recognize the label and present it to the user, therefore allowing the user to identify the purpose of the control. The label may also be used to include text or a text symbol indicating that input is required.
```

Comment:
"Using the appropriate technology-specific techniques for technologies for associating labels with interactive controls allows assistive technology to recognize the label and present it to the user, therefore allowing the user to identify the purpose of the control."

"technology-specific techniques for technologies for associating" is rather a mouthful. that "for technologies" should probably be removed - "technology-specific techniques for associating"

But at a much more fundamental level ... isn't this whole bit about associating things in markup/making the visual relationship clear in code ... so a 1.3.1 issue? I'd remove this, or rather clarify that this part is addressed by 1.3.1, and NOT by 3.3.2